client/a8c-for-agencies/sections/purchases/licenses/license-actions/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button, Gridicon } from '@automattic/components';
```

Comment:
Can we use the button from @wordpress/components?

```suggestion
import { Button } from '@wordpress/components';
```

Reply:
@yashwin, I attempted to change this, but there are a couple of alignments that also need fixing if we decide to change this now. I didn't want to bloat the change here. Can we do this in a separate PR?

Reply:
Sounds good, James!